08-rails/html-forms.md

```diff
@@ -82,19 +82,19 @@ Give it a try. It broke, right? We haven't taught our app how to make this work!
 Let's make our form a little bit smarter. We can submit nested data by manipulating the `name` attribute.
 
 ```html
-  <input type="text" name="entry[title]" value="" id="author">
-  <input type="text" name="entry[description]" value="" id="description">
+  <input type="text" name="book[title]" value="" id="title">
+  <input type="text" name="book[author]" value="" id="author">
 ```
 
-In Rails this is the way we _model_ data objects. Imagine a `Peep` class where the initialize method is coded to assign attributes from a hash. We would then pass the inner hash to the `new` method and all of our attributes would be assigned.
+In Rails this is the way we _model_ data objects. We will expand upon this idea once we learn about Active Record, but until then, imagine a `Book` class where the initialize method is written to assign attributes from a hash. We would then pass the inner hash to the `new` method and all of our attributes would be assigned.
 
 ```ruby
-class Entry
+class Book
   def initialize(options = {})
     @title = options[:title]
-    @description = options[:description]
+    @author = options[:author]
   end
 end
 
-entry = Entry.new(params["Interesting Title"])
+book = Book.new(title: params[:title], author: params[:author])
```

Comment:
Since we're talking about nested params here we should actually use them:
```ruby
book = Book.new(title: params[:book][:title], author: params[:book][:author])
```

We should also have a comment indicating that this code would go in `app/controllers/books_controller.rb`.